leaf-ui/rappid-extensions/ActorInspector.js

```diff
@@ -2,37 +2,42 @@
 var ENTER_KEY = 13;
 
 var ActorInspector = Backbone.View.extend({
-
-        className: 'element-inspector',
-
+        model: joint.shapes.basic.Actor,
+        
+        initialize: function() {
+            this.model.on('change:actorType', this.changeLine, this);
+        },
         template: [
+            '<script type="text/template" id="item-template">',
+            '<div class="inspector-views">',
             '<label>Actor name</label>',
-            '<textarea class="cell-attrs-text" maxlength=100></textarea>',
+            '<textarea class="cell-attrs-text" maxlength=100> <%= actorName %> </textarea>',
             '<label> Actor type </label>',
             '<select class="actor-type">',
             '<option value=A> Actor </option>',
             '<option value=G> Agent </option>',
             '<option value=R> Role </option>',
-            '</select>'
+            '</select>',
+            '</div>',
+            '</script>'
         ].join(''),
 
+    
         events: {
             'keyup .cell-attrs-text': 'nameAction',
+            'change .actor-type': 'updateType', 
+            'clearInspector .inspector-views': 'removeView',
+
         },
 
         /**
          * Initializes the element inspector using previously defined templates
          */
-        render: function(cell) {
-
-            this.cell = cell;
-
-            // Save actor here
-            this.actor = model.getActorByID(cell.attributes.nodeID);
-
+        render: function() {
             // If the clicked node is an actor, render the actor inspector
-            this.$el.html(_.template(this.template)());
-            this.$('.cell-attrs-text').val(this.actor.nodeName);
+            this.$el.html(_.template($(this.template).html())(this.model.toJSON()));
+            var actorType = this.model.get('actorType');
+            $('.actor-type').val(actorType);
```

Comment:
You should be able to do this in the script as well - see lines 19-24 of AnalysisInspector.js